index.js

```diff
@@ -157,25 +158,38 @@ app.delete('/users', (req, res) => {
         res.status(400).send(error.message);
     }
     res.status(200).send('User successfully deleted.');
+});
 
 //DONATION METHODS
 
 app.get('/donations', (req, res) => {
-    res.status(200).json(ideasDonationSum(req.body.ideaId));
+    res.status(200).json(getTotalSumDonatedForIdea(req.body.ideaId));
 });
 
 app.get('/donations/sumDifference', (req, res) => {
-    res.status(200).json(ideasSumDifference(req.body.ideaId));
+    res.status(200).json(getIdeasSumDifference(req.body.ideaId));
 });
 
 app.post('/donations', (req, res) => {
     try {
-        createDonation(
-            req.body.firstName,
-            req.body.sum,
-            req.body.userId,
-            req.body.ideaId
-        );
+        let searchResults = getUser(req.body.userId);
+        let user = searchResults[1];
```

Comment:
`searchResults` should not be an array here, because `getUser` should return a user object, not an array

Reply:
Done